Cauldron/Assets/Scripts/UI/Menu/SettingsMenu.cs

```diff
@@ -214,5 +238,11 @@ private void ResetGameData()
             mainMenu.ResetGameData();
             CloseResetDialogue();
         }
+        
+        private void OpenAutoCooking() => 
```

Comment:
It would be better to get all these methods related to cooking together, so that they would be easily readable one after another.
Google "newspaper code structure"